src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -30,6 +32,8 @@ function LineVis(props: Props): ReactElement {
     showGrid = true,
     scaleType = ScaleType.Linear,
     abscissas = range(dataArray.size),
+    abscissaLabel,
+    ordinateLabel,
```

Comment:
What do you think of what I mentioned in the chat: combining `abscissas` and `abscissaLabel` in a single prop called `abscissa` with type `{ label: string; values: number[] }`?

Reply:
I think it is a nice idea :smile_cat: